src/notifications/use-push-notifications.tsx

```diff
@@ -127,6 +124,14 @@ export const NotificationContextProvider: React.FC = ({children}) => {
       checkPermissions();
   }, [pushNotificationsEnabled, checkPermissions, authStatus]);
 
+  useEffect(() => {
+    (async function () {
+      const token = await messaging().getToken();
+      if (!token) return;
```

Comment:
Do we know in what situations the token can be `undefined`. As it is now we just try once at startup, and if the token is `undefined` then it will never be retried until a new app start.

Reply:
The promise always returns a nil as far I can see, so then I guess is because an explicit exception going on.